packages/StatusTracker/src/components/Point/Point.js

```diff
@@ -0,0 +1,172 @@
+  }
+
+  function getTrigger(handler, a11yAttributes) {
+    if (kind === kinds.CURRENT && extractedOverflowMenu) {
+      const { children: overflowMenuChildren } = extractedOverflowMenu.props;
+      const { "OverflowMenu.Trigger": extractedOverflowMenuTrigger } = extractChildren(overflowMenuChildren, [
+        "OverflowMenu.Trigger",
+      ]);
+
+      return (
+        <sc.Point
+          onMouseOver={handleTooltipOpen}
+          onMouseOut={handleTooltipClose}
+          onFocus={handleTooltipOpen}
+          onBlur={handleTooltipClose}
+          kind={kind}
+          data-pka-anchor="status-tracker.point"
+          {...a11yAttributes}
+        >
+          {React.cloneElement(
+            extractedOverflowMenu,
+            {
+              isOpen: isOverflowMenuOpen,
+              onClose: handleOverflowMenuClose,
+              "data-pka-anchor": "status-tracker.point.overflow-menu",
+            },
+            <>
+              {extractedOverflowMenuTrigger ? (
+                React.cloneElement(
+                  extractedOverflowMenuTrigger,
+                  {
+                    buttonType: OverflowMenu.Trigger.types.button.SIMPLE,
+                    isDropdown: true,
+                    kind: Button.types.kind.PRIMARY,
+                    onClick: handleOverflowMenuOpen,
+                    "data-pka-anchor": "status-tracker.point.overflow-menu.trigger",
+                  },
+                  name
+                )
+              ) : (
+                <OverflowMenu.Trigger
+                  buttonType={OverflowMenu.Trigger.types.button.SIMPLE}
+                  isDropdown
+                  kind={Button.types.kind.PRIMARY}
+                  onClick={handleOverflowMenuOpen}
+                  data-pka-anchor="status-tracker.point.overflow-menu.trigger"
+                >
+                  {name}
+                </OverflowMenu.Trigger>
+              )}
+
+              {extractedOverflowMenuTrigger
+                ? overflowMenuChildren.filter(child => !child.type || child.type.displayName !== "OverflowMenu.Trigger")
+                : overflowMenuChildren}
+            </>
+          )}
+        </sc.Point>
+      );
+    }
+
+    return (
+      <sc.Point
+        onMouseOver={handleTooltipOpen}
+        onMouseOut={handleTooltipClose}
+        onFocus={handleTooltipOpen}
+        onBlur={handleTooltipClose}
+        kind={kind}
+        as={RawButton}
+        data-pka-anchor="status-tracker.point"
+        {...a11yAttributes}
+      >
+        {" "}
+      </sc.Point>
+    );
+  }
+
+  return (
+    <sc.PointWrapper aria-current={kind === kinds.CURRENT} kind={kind} {...moreProps}>
+      <sc.Popover isEager isDark isOpen={isTooltipOpen}>
+        <Popover.Trigger>{getTrigger}</Popover.Trigger>
+        <Popover.Content>
+          <sc.PopoverCard hasOnlyName={!!name && !!description}>
+            {name && <sc.NameInTooltip data-pka-anchor="status-tracker.point.name">{name}</sc.NameInTooltip>}
```

Comment:
this will fail on `"" && <sc.NameInTooltip data-pka-anchor="status-tracker.point.name">{name}</sc.NameInTooltip>` it will print the `""`. I know is a little more typing but going with the ternary is slighter safer. or change it to `Boolean(name)`